test/support/BaseSpec.scala

```diff
@@ -73,7 +72,7 @@ class BaseSpec
   )
   val validSessionToken: String = Session.generateToken("1234")
   val invalidSessionToken: String = Session.generateToken("invalid-user")
-  implicit val apiConnectorMonitoring: ApiConnectorMonitoring = TestApiConnector
+  implicit val apiConnectorMonitoring: MessagingServiceMonitoring = TestMessagingService
   lazy implicit val testTimeProvider: TimeProvider = () => LocalDateTime.of(2023, Month.FEBRUARY, 28, 1, 1, 1)
```

Comment:
could the variable be renamed to `messagingServiceMonitoring` for consistency

Reply:
Renamed.